recipes/perf/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class Perf(ConanFile):
+    name = "perf"
+    description = "Linux profiling with performance counters"
+    topics = ("linux", "profiling")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://perf.wiki.kernel.org/index.php"
+    license = "GPL-2.0"
+    settings = "os", "compiler", "build_type", "arch"
+
+    _source_subfolder = "source_subfolder"
```

Comment:
```suggestion
    @property
    def _source_subfolder(self):
        return "source_subfolder"
```

Reply:
why is this better?

Reply:
It's a pattern that we use here. It's not better.

Reply:
grepping shows there are 68 recipes with this shorter version, if it's not a blocker I prefer to stick to it.

Reply:
>  why is this better?

because is read-only. Also, try to keep the same pattern.